QuickNII_functions.py

```diff
@@ -17,13 +16,14 @@ def pd_to_quickNII(results, orientation='coronal', filename='Download'): ##conve
             results = results.sort_values('oz')
         ##Explicitly confirm all filenames are Strings
         results['Filenames'] = results['Filenames'].astype(str)
+        df = results
```

Comment:
good catch, Im going to remove this line and edit future references to df to instead reference results